pkgs/by-name/at/atools/package.nix

```
@@ -0,0 +1,47 @@
+{ fetchFromGitHub
+, pkg-config
+, stdenv
+, lib
+# Package dependencies
+, qt5
+}:
+
+stdenv.mkDerivation rec {
+  pname = "atools";
+  version = "4.0.6";
+
+  src = fetchFromGitHub {
+    owner = "albar965";
+    repo = "atools";
+    rev = "v${version}";
+    sha256 = "sha256-ieL7cGqXNmGMyTf8tXCSmwQFwyHyy8NwGJ+3Zg92nmM=";
```

```suggestion
    hash = "sha256-ieL7cGqXNmGMyTf8tXCSmwQFwyHyy8NwGJ+3Zg92nmM=";
```

https://nixos.org/manual/nixpkgs/unstable/#fetchfromgithub